src/qt/forms/debugwindow.ui

```diff
@@ -249,6 +249,29 @@
         </widget>
        </item>
        <item row="9" column="0">
+        <widget class="QLabel" name="label_14">
```

Comment:
nit:

```suggestion
        <widget class="QLabel" name="labelLocalAddresses">
```

Reply:
thanks for the checks & ACK @shaavan . I kept the label_14 in accordance in most of other fields which had the same pattern.

regarding the listening port, you can test with something like (for testnet):

```
testnet=1
blocksonly=1
[test]
#prune=1000 # this has no effect during syncing
externalip=100.200.100.10
rpcbind=127.0.0.1
rpcallowip=0.0.0.0/0
rpcuser=user
rpcpassword=pass
```

as your bitcoin.conf to force a listening IP.